app/controllers/logs_controller.rb

```diff
@@ -59,6 +59,20 @@ def show
     @color = Static::COLOR
   end
 
+  def destroy
```

Comment:
__[rails_best_practices]__
* move model logic into model (log use_count > 4) [:ref](http://rails-bestpractices.com/posts/2010/07/21/move-model-logic-into-the-model/)